assets/js/googlesitekit/modules/datastore/settings.js

```diff
@@ -25,7 +25,10 @@ import invariant from 'invariant';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-const { createRegistrySelector } = Data;
+const { createRegistrySelector, createRegistryControl } = Data;
+import { STORE_NAME as CORE_MODULES } from './constants';
```

Comment:
No alias should be used here since it is using its own store name; we only alias store names when imported from other stores when in the context of a store (or store partial in this case).

```suggestion
import { STORE_NAME } from './constants';
```